components/Wallet.tsx

```diff
@@ -63,15 +63,11 @@ const Wallet: React.FC<{}> = () => {
 						whiteSpace: "nowrap",
 						textAlign: "center",
 						letterSpacing: "1.2px",
+						ml: "10px",
 					}}
 				>
 					{isWalletConnected
-						? selectedAccount?.address.slice(0, 9) +
-						  "..." +
-						  selectedAccount?.address.slice(
-								selectedAccount?.address.length - 4,
-								selectedAccount?.address.length
-						  )
+						? selectedAccount?.meta?.name.substring(0, 17)
```

Comment:
for future case, you can use the CSS rule `text-overflow: ellipsis` to truncate and add `...` to the text when it's overflow

Reply:
O awesome, I'll change that in this PR